src/main/java/edu/cornell/kfs/kns/datadictionary/validation/fieldlevel/IntegerValidationPattern.java

```diff
@@ -0,0 +1,22 @@
+package edu.cornell.kfs.kns.datadictionary.validation.fieldlevel;
```

Comment:
since this object extends FieldLevelValidationPattern and that is in the KRAD package, should this object also be in KRAD?

Reply:
Even though the FieldLevelValidationPattern superclass is in the KRAD package, the majority of its implementation subclasses are in the KNS package. I put this class in the KNS package for consistency with those other implementations, but I'm fine with relocating it to the KRAD package for consistency with the superclass instead. I'll proceed with that change.